### DISCUSSION TITLE
为啥我的安装后像是没网络，加载不出来任何东西。

### DISCUSSION THREAD

**Discussion by fanceyh** [Richasy/Bili.Uwp#709] [问答] [answered]
RT：
为啥我的安装后像是没网络，加载不出来任何东西。
![图片](https://user-images.githubusercontent.com/48702727/147022976-679ff521-8cc8-4ffc-b35a-4c307689da36.png)

**Comment by huuhghhgyg** [comment, marked as answer]
设置 - 网络和Internet - WLAN - 选择能够使用你的WLAN数据的应用
把哔哩的开关打开就ok😂

**Reply by fanceyh** [reply]
按你的方法解决了，谢谢答主。

**Reply by Bradford-Chen** [reply]
设置里没有这个选项怎么办
